tools/sdk-generation-pipeline/documents/docker/build-docker-image.md

```diff
@@ -0,0 +1,10 @@
+If you want to rebuild the docker image, you only can do it in your local. It's suggested to create a release pipeline to build it. (Previously we find that sometimes it will be timeout because the pipeline costs much time in building and pushing docker image, but now the agent pool should be very fast.)
```

Comment:
@maririos @praveenkuttappan I agree that we should create a pipeline to generate and publish this. Probably worth a tracking item.

Reply:
Nevermind it looks like @dw511214992 already did the work in https://github.com/Azure/azure-sdk-tools/pull/5589. Thanks Wei.